sdk/sdk/src/main/java/org/apache/carbondata/sdk/file/CarbonWriterBuilder.java

```diff
@@ -194,6 +194,7 @@ public CarbonWriterBuilder uniqueIdentifier(long timestamp) {
    *                h. quotechar
    *                i. escapechar
    *                j. fileheader
+   *                k. line_separator
```

Comment:
need to update sdk-guide.md document file also

Reply:
> need to update sdk-guide.md document file also

Since line separator has no usage in sdk, remove the changes.